Cartfile

```diff
@@ -1,2 +1,2 @@
-github "ReactiveCocoa/ReactiveObjC" ~> 3.1.0
-github "ReactiveCocoa/ReactiveSwift" ~> 4.0.0
+github "ReactiveCocoa/ReactiveObjC" "27a7254837bb01122b51aac7c4c3961973351ca0"
```

Comment:
Actually, we should get an actual release here

Reply:
I just released 3.1.1. Please update the Cartfile with that.
https://github.com/ReactiveCocoa/ReactiveObjC/releases/tag/3.1.1